aicsimageio/readers/czi_reader.py

```diff
@@ -395,7 +396,7 @@ def _create_dask_array(self, czi: CziFile) -> xr.DataArray:
             # Get pixel type and catch unsupported
             pixel_type = PIXEL_DICT.get(czi.pixel_type)
             if pixel_type is None:
-                raise TypeError(f"Pixel type: {pixel_type} is not supported.")
+                raise TypeError(f"Pixel type: {czi.pixel_type} is not supported.")
 
             # Add delayed array to lazy arrays at index
             lazy_arrays[np_index] = da.from_delayed(
```

Comment:
handling the "invalid" pixeltype allows this part of the code to go on, by creating a dask array with a realistic dtype.  I am then just totally dropping all that work with the other changes.

Reply:
I suppose I could try to short circuit things way up here, too but it was vastly easier to do it at the end of it all.